src/ucp/tag/rndv.c

```diff
@@ -420,8 +420,10 @@ ucs_status_t ucp_tag_send_start_rndv(ucp_request_t *sreq)
 void ucp_tag_rndv_cancel(ucp_request_t *sreq)
 {
     if (!(sreq->send.ep->flags & UCP_EP_FLAG_REMOTE_CONNECTED)) {
-        if (sreq->flags & UCP_REQUEST_FLAG_RNDV_RTS_SENT) {
+        if (!(sreq->flags & UCP_REQUEST_FLAG_RNDV_RTS_SENT)) {
```

Comment:
minor: maybe invert if/else? positive conditions are simpler for reading

Reply:
good idea, done